src/main/flight/pid.h

```diff
@@ -72,6 +72,7 @@
 
 #ifdef USE_WING
 #define TPA_LOW_RATE_MIN INT8_MIN
+#define TPA_GRAVITY_MAX 5000
```

Comment:
Now that I understand what this value does, I was probably a bit optimistic with such a high value 😀 1000 should be plenty.